requirements/edx/custom.txt

```diff
@@ -1,3 +1,6 @@
 # RACCOONGANG
 sentry-sdk==0.7.8
 # RACCOONGANG
+
+# UCDC integration
+-e lms/lib/ucdc_edx_api_plugin
```

Comment:
I still no agree, local.in looks more attractive to put local libs with the own setup files.

Reply:
Perhaps local is a more understandable and convenient place. But I had a conversation on this subject with DevOps engineer Oleksii Gnedin. He made it clear to me from which files the environment on Staging or Production is deployed. Always from .txt and not from local.txt (in) for sure. I have no expertise here and I trust my colleagues. Alex was very convincing 🙂